.gometalinter.json

```diff
@@ -5,7 +5,6 @@
   "EnableGC": true,
   "Enable": [
     "structcheck",
-    "staticcheck",
```

Comment:
@dmcgowan why the removal of static check?

Reply:
static check is failing on this branch and it is not requirement. It will remain on the master branch. The number of fixes related to static check is too large to backport imo.